src/video_core/texture_cache/texture_cache.h

```diff
@@ -782,6 +897,115 @@ class TextureCache {
         return {};
     }
 
+    bool EnabledRescaling() const {
+        return enable_resolution_scaling;
+    }
+
+    bool IsResScannerEnabled() const {
```

Comment:
```suggestion
    bool IsResolutionScannerEnabled() const {
```

`Res` is easily confused with other names (e.g. Resource), It's also inconsistent with naming elsewhere in the codebase (IsResolutionScalingEnabled, etc)